bittensor/subtensor.py

```diff
@@ -1378,7 +1372,7 @@ def get_transfer_fee(
 
     def _do_transfer(
         self,
-        wallet: "bittensor.wallet",
+        wallet: bittensor.wallet,
```

Comment:
There is inconsistency on `bittensor.wallet`. 

sometimes its with quotes, and sometimes without. 
eg, ln 1375 vs 1487

Reply:
Yes, you are right. Today it was decided that we will not adhere to a specific rule (in quotes or not) for the annotation.
But in order to maintain the consistency of the refactoring, I tried to fix it throughout the entire module. If I missed it somewhere, please feel free to show it. Thank you!